docs/DEPLOYMENT.md

```diff
@@ -211,64 +262,163 @@ After the deployment is done successfully, we need to expose the URL to access t
+
+3. To create an agent, you can use the `/system/user/agent`. You will need `agent:create` scope in your JWT which has to be added in config.yaml
+
+    ```
+    curl -X PUT --header "Content-Type: application/json" \
+        -H "Authorization: <access-token>" \
+        --data '{"name":"catalog-refresh-agent","scopes": ["catalog:refresh"]}' \
+        <api-route>/system/user/agent
+    ```
+    Replace `<access-token>` with your JWT, you can create an agent with any required scopes. In this case we need the agent with `catalog:refresh`. You can give any name to agent.
+
+    This will gives an output as:
+    ```
+    {"token":"agent jwt token"}
+    ```
+    A token will be returned with the requested scopes. You can check using the jwt decoder. Use this token for the catalog refresh cron job.
+
+4. Navigate to `api/config/99-post-deploy` and edit `34-catalog-refresh-secret.yaml`. Set the Hub Token
+
+   ```yaml
+   apiVersion: v1
+   kind: Secret
+   metadata:
+     name: catalog-refresh
+     namespace: tekton-hub
+   type: Opaque
+   stringData:
+     HUB_TOKEN: hub token
+   ```
+    Set the token created in last step as Hub token.
+
+4. Make sure you are in `api` directory and run the below command. 
+
+    ```
+    kubectl apply -f config/99-post-deploy/34-catalog-refresh-secret.yaml \ 
+                -f config/99-post-deploy/35-catalog-refresh-cronjob.yaml 
+    ```
+    The cron job is configured to run every 30 min, you can change the interval in `config/99-post-deploy/35-catalog-refresh-cronjob.yaml`.
+    
+## Adding New Users in Config
+
+By default when any user login for the first time, they will have only default scope even if they are added in config.yaml. To get additinal scopes, make sure the user has logged in once.
+
+Now, we need to refresh the config. To do that do a POST API call.
+
+```
+curl -X POST -H "Authorization: <access-token>" \
+    --header "Content-Type: application/json" \
+    --data '{"force": true} \
+  <api-route>/system/config/refresh
+```
+Replace `<access-token>` with your JWT token. you must have `config-refresh` scope to call this API. 
+
+## Troubleshooting
+
+Case 1: UI is not showing resources but catalog refresh is successful
+  If you have insecure connection the UI might not be able to hit the API.
+  So, hit the API URL in your browser `https://<api-url>` once you get the response as `status: ok`, refresh you UI.
+
+Case 2: UI is not showing resources but catalog refresh is successful (Console Error)
+  Verify catalog refresh is successful by hitting the `/v1/resources` API using the API URL.
+  If is returning resources than check for console errors on UI. 
+  Right click -> Inspect or (Ctrl + Shift + I) (For Chrome)
+  Then click on console tab.
+  If there is cors error then check the UI config map and the URL you have added.
+  The URL should be for example `https://api.hub.tekton.dev` and not `https://api.hub.tekton.dev/` there shouldn't be `/` at the end.
```

Comment:
Can you please add these in points as in preview mode they are being shown as one paragraph?